test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/persistence/ExternalDatabaseIntegrationTest.java

```diff
@@ -0,0 +1,79 @@
+package org.kie.cloud.integrationtests.persistence;
+
+import static org.junit.Assume.assumeTrue;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.junit.Before;
+import org.junit.Test;
+import org.kie.cloud.api.DeploymentScenarioBuilderFactory;
+import org.kie.cloud.api.deployment.constants.DeploymentConstants;
+import org.kie.cloud.api.scenario.KieServerWithExternalDatabaseScenario;
+import org.kie.cloud.common.provider.KieServerClientProvider;
+import org.kie.cloud.integrationtests.AbstractCloudIntegrationTest;
+import org.kie.cloud.maven.MavenDeployer;
+import org.kie.cloud.maven.constants.MavenConstants;
+import org.kie.server.api.model.KieContainerResource;
+import org.kie.server.api.model.KieServerInfo;
+import org.kie.server.api.model.ReleaseId;
+import org.kie.server.api.model.ServiceResponse;
+import org.kie.server.api.model.instance.ProcessInstance;
+import org.kie.server.api.model.instance.TaskSummary;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.client.ProcessServicesClient;
+import org.kie.server.client.UserTaskServicesClient;
+
+public class ExternalDatabaseIntegrationTest extends AbstractCloudIntegrationTest<KieServerWithExternalDatabaseScenario> {
+
+    private KieServicesClient kieServerClient;
+
+    @Override protected KieServerWithExternalDatabaseScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
+        assumeTrue(isExternalDatabaseAllocated());
+        return deploymentScenarioFactory.getKieServerWithExternalDatabaseScenarioBuilder()
+                .withExternalMavenRepo(MavenConstants.getMavenRepoUrl(), MavenConstants.getMavenRepoUser(), MavenConstants.getMavenRepoPassword())
+                .build();
+    }
+
+    @Before
+    public void setUp() {
+        kieServerClient = KieServerClientProvider.getKieServerClient(deploymentScenario.getKieServerDeployment());
+    }
+
+    @Test
+    public void testKieServerRunning() {
```

Comment:
Do we need this test here? What does it bring?

Reply:
It just test that Kie server is running and waiting for requests.